src/dom/dom.js

```diff
@@ -2063,6 +2063,19 @@ p5.Element.prototype.size = function(w, h) {
  * </code></div>
  */
 p5.Element.prototype.remove = function() {
+  // stops all audios/videos and detach all devices like microphone/camera
+  // used as input/output for audio/video.
+  if (this instanceof p5.MediaElement) {
+    var stream = this.elt.srcObject;
+    var tracks = stream.getTracks();
```

Comment:
can you change these to `const`?

Reply:
this could be combined into one line to condense:
`let tracks = this.elt.srcObject.getTracks();`